docker-compose.yml

```diff
@@ -23,4 +23,4 @@ services:
     ports:
       - "$DB_PORT_HOST:$DB_PORT_GUEST"
 volumes:
-  my-datavolume:
+  my-datavolume: {}
```

Comment:
What does this change effect? Is this your prettifier tuning files you don't intend it to?

Reply:
This change came from when I was trying to gain connection to mysql and through my research I saw other files from people that had that there so I was trying different methods to gain access.